### ISSUE TITLE
Change size of graphic

### ISSUE THREAD

**Issue by Munzu** [IllDepence/anki_add_pitch_plugin#13]
Hello,

first of all, great add-on, thank you so much for your hard work.

How can I change the size of the graphic? 

I was messing around and this is the only way I got it to work:
![image](https://user-images.githubusercontent.com/31477160/117299746-2cc02d80-ae79-11eb-8338-933477b8fffa.png)

I can set the width as a percentage of the screen but for some reason, I also have to give `height="` specifically or else it doesn't work. I can't leave it out, or give it its own percentage, or give `height=""`. 
It has to be `height="`. 

Surely, this isn't the proper way to do it, right?
I have no experience in web design so any help would be much appreciated.

**Comment by IllDepence** [comment]
Hey,

glad you like the add-on and nice to see it being used in a dark mode setting.

**TLDR**: `height="auto"` should be a cleaner way to achieve what you're doing. Even cleaner should be something like 

```
svg.pitch {
width: 30%;
height: auto;
}
```

in the *Styling* tab instead of *Back Template*.

**Some background**

> It has to be height=".
> Surely, this isn't the proper way to do it, right?

That's curious indeed. I presume giving an incomplete definition for the height triggers some default behavior (like `auto`).

What you insert into your card with `{{C}}` already is an `<svg>` element with some width and height set. E.g.: `<svg width="137px" height="75px" viewBox="0 0 137 75">`. You currently put another `<svg>` around that, thereby restricting the contained element's size. If you, however, use the *Styling* tab as described above, the CSS rules for width and height you define are directly applied to the original SVG.

Hope this helps. (:

**Comment by Munzu** [comment]
That makes sense, thank you so much! `svg.pitch` works like a charm.

It might be a good idea to include this info on the AnkiWeb page. I can imagine a lot of other people wanting to resize the graphics.

**Comment by cocowash** [comment]
@IllDepence is it possible to change the size of the graphic for Anki droid? With this tweak the font is readable, but I find it too small on my phone.
